jsapp/js/components/activity/formActivity.scss

```diff
@@ -0,0 +1,91 @@
+@use 'scss/sizes';
+@use 'scss/mixins';
+@use 'scss/breakpoints';
+@use 'scss/colors';
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  background-color: colors.$kobo-white !important;
+
+  .header {
+    width: 100%;
+    display: flex;
+    gap: sizes.$x8;
+    align-items: center;
+    padding: 0 sizes.$x20;
+
+    h1 {
+      color: colors.$kobo-gray-800;
+    }
+
+    .header-actions {
+      flex: 1;
+      display: flex;
+      gap: sizes.$x8;
+      justify-content: flex-end;
+    }
+
+    .filter-select {
+      width: sizes.$x150;
+    }
+  }
+
+  .table-container {
+    flex: 1;
+    overflow: auto;
+    padding: sizes.$x20;
+
+    .loading {
+      width: 100%;
+      height: 100%;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    td {
+      color: colors.$kobo-gray-700;
```

Comment:
I would say that any override to `UniversalTable` styles should really be incorporated into `UniversalTable` itself. We are generally trying to have the least amount of overrides to existing components. It is better to unify the looks even if this means not following designs in 100%.

But in this case, I think that Figma file says it should be gray-800, so the default text color. Remove this and either do nothing, or add `color: colors.$kobo-gray-800` to `universalTable.module.scss` :)